terraform/module/lambda.tf

```diff
@@ -15,6 +15,14 @@ resource "aws_lambda_function" "analyzer" {
 
   runtime = "nodejs16.x"
 
+  dynamic "vpc_config" {
```

Comment:
@ldcorentin isn't there a way to define this object only once and reuse it for all functions?

Reply:
@alexcasalboni To my knowledge this is not possible